docs/interactions/Slash_Commands.md

```diff
@@ -627,6 +627,46 @@ And, done! The JSON looks a bit complicated, but what we've ended up with is a s
 
 ![A command with grouped subcommands and parameters. It says /permissions user get with arguments for a user and a channel.](command-with-groups-subcommands-parameters.png)
 
+## Permissions
+
+Need to keep some of your Slash Commands safe from prying eyes, or only available to the right people? Slash Commands support permission overwrites for all your commands. For both guild _and_ global commands, you can enable or disable a specific user or role in a guild from using a command.
+
+> info
+> For now, disabled commands still show up in the command picker, but are unable to be used.
+
+You can also set a `default_permission` on your commands if you want them to be disabled by default when your app is added to a new guild. Setting `default_permission` to `false` will disallow _anyone_ in a guild from using the command--even Administrators--unless a specific overwrite is configured.
+
+For example, this command will not be usable by anyone in any guilds by default:
+
+```json
+{
+    "name": "permissions_test",
+    "description": "A test of default permissions",
+    "default_permission": false
+}
+```
+
+To enable it just for a moderator role:
+
+```py
+MODERATOR_ROLE_ID = "173547401905176999"
+url = "https://discord.com/api/v8/applications/<my_application_id>/guilds/<my_guild_id>/commands/<my_command_id>/permissions"
+
+json = {
+    "id": MODERATOR_ROLE_ID,
+    "type": 1,
+    "permission": True
+}
+
+headers = {
+    "Authorization": "Bot 123456"
+}
+
+r = requests.put(url, headers=headers, json=json)
```

Comment:
The example is wrong, the json should be:
```py
json = {
    "permissions": [
        {
             "id": MODERATOR_ROLE_ID,
             "type": 1,
             "permission": True
        }
    ]
}
```

Reply:
TY ty